Makefile

```
@@ -367,3 +367,10 @@ sbom:
 	docker run -it --rm \
 		-v "$$PWD:/edgex-go" -v "$$PWD/sbom:/sbom" \
 		spdx/spdx-sbom-generator -p /edgex-go/ -o /sbom/ --include-license-text true
+
+docker-fuzz:
+	docker build -f Dockerfile.fuzz -t fuzz-edgex-go:latest .
+
+fuzz-test:
+# not joining the edgex-network due to swagger file url pointing to localhost for fuzz testing in the container
+	docker run --net host --rm -v "$$PWD/fuzz_results:/fuzz_results" fuzz-edgex-go:latest
```

This should fail when requested changes are made to Dockerfile?
We need a target for each service, starting with `core-command`
Then as you work out the other services, you add targets for each of them. Once all are ready we added a fuzz-test-all target.
```suggestion
	docker run --net host --rm -v "$$PWD/fuzz_results:/fuzz_results" fuzz-edgex-go:latest core-command /restler-fuzzer/openapi/core-command.yaml
```